components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1361,8 +1361,7 @@ void JsonRpcService::GetERC721TokenBalance(
     const std::string& chain_id,
     GetERC721TokenBalanceCallback callback) {
   const auto eth_account_address = EthAddress::FromHex(account_address);
-  auto network_url = GetNetworkURL(prefs_, chain_id, mojom::CoinType::ETH);
-  if (eth_account_address.IsEmpty() || !network_url.is_valid()) {
```

Comment:
ugh I should catch it in the previous review since network_url is not actually used before this PR.